domains/pallets/transporter/src/lib.rs

```diff
@@ -17,7 +17,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 #![forbid(unsafe_code)]
-#![warn(rust_2018_idioms, missing_debug_implementations)]
```

Comment:
Why removing lint, was it not possible to fix?

Reply:
Simply adding a `#[derive(Debug)]` can fix it but I don't think this lint is necessary.